src/Settings/Equations/ions.cpp

```diff
@@ -549,7 +550,7 @@ void SimulationGenerator::ConstructEquation_Ions(
 			if (init_equil[iZ] != 0)
 				continue;
 
-			for (len_t Z0 = 1; Z0 <= ih->GetZ(iZ); Z0++, ionOffset++) {
+			for (len_t Z0 = 0; Z0 <= ih->GetZ(iZ); Z0++, ionOffset++) {
```

Comment:
This modification (``1`` -> ``0``) is perhaps not necessary? For ``Z0=0``, the product on line 555 is always 0, so ``Z0=0`` will never provide a contribution?

Reply:
You're right. It was an attempt to update `ionOffset` correctly, but I realised I forgot to update it for the equilibrium ions. I believe I fixed this in commit 53d9839 (as well as avoiding the `Z0=0` case).